### PR TITLE
Adding publish JUnit results option to Ant task and PS implementation

### PR DESCRIPTION
**Pull request by madhurig**

### CODE REVIEW

File: Tasks/ANT/ant.ps1 [hunk 1]
```diff
@@ -38,6 +42,27 @@ if($jdkVersion -and $jdkVersion -ne "default")
...
+    }
+    else
+    {
+        Write-Verbose "Calling Publish-TestResults"
+        Publish-TestResults -TestRunner "JUnit" -TestResultsFiles $matchingTestResultsFiles -Context $distributedTaskContext
+    }    
+}
+else
+{
+    Write-Host "Option to publish JUnit Test results produced by Ant build was not selected and is being skipped."
```
Comment (jpricketMSFT) [inline Tasks/ANT/ant.ps1]: This seems like a verbose statement.

File: Tasks/ANT/task.json [hunk 2]
```diff
@@ -45,7 +50,25 @@
...
+            "type": "filePath",
+            "label": "Test Results Files",
+            "defaultValue": "**\\TEST-*.xml",
+            "required": true,
+            "groupName":"junitTestResults",
+            "helpMarkDown": "Test results files path.  Wildcards can be used.  For example, `**\\*TEST-*.xml` for all xml files whose name starts with TEST-."
+        },            
         {
             "name":"jdkVersion",
             "type":"pickList",
```
Comment (jpricketMSFT) [inline Tasks/ANT/task.json]: You need to build to get the task.loc.json file generated. It should be checked in with the changes.

File: Tasks/ANT/ant.ps1 (hunk 1 above)
Comment (madhurig) [inline Tasks/ANT/ant.ps1]: I wanted the user to see it in the output to make it clear why test publishing was being skipped.

Comment (jpricketMSFT) [inline Tasks/ANT/ant.ps1]: okay, it just seems odd to print the statement when the user set the option "publishJUnitResults" to false.

File: Tasks/ANT/task.json (hunk 2 above)
Comment (madhurig) [inline Tasks/ANT/task.json]: Thanks for clarifying, both loc json and resource strings are committed.

File: Tasks/ANT/ant.ps1 (hunk 1 above)
Comment (madhurig) [inline Tasks/ANT/ant.ps1]: You convinced me, changed it to verbose statement.

**Pull request merged**